src/gbd.c

```diff
@@ -1521,15 +1523,15 @@ decode_noop_cmd (void)
             gfxd_printf("gsDPNoOpCallBack(0x%08X, 0x%04X),", noop_data->u, noop_data1->u);
             break;
         case 7:
-            gfxd_printf("gsDPNoOpOpenDisp(");
+            gfxd_printf(VT_GREEN "gsDPNoOpOpenDisp" VT_RESET "(");
```

Comment:
Names need updating to use `CLR`

Reply:
wow nice catch, I have no clue how I pulled that off